scripts/updatecli/updatecli_config.tpl

```diff
@@ -81,6 +86,16 @@ sources:
       versionFilter:
         kind: regex
         pattern: '{{ .adw.version }}(.(\d+))+{{ .version_pattern }}'
+  agsAmp:
+    name: AGS AMP {{ .ags.version }}.x
+    kind: gittag
+    scmid: acsEntRepo
```

Comment:
https://github.com/Alfresco/governance-services seem to have more recent/consostent tags